CHANGELOG.md

```diff
@@ -1,8 +1,9 @@
-## 2.9.1 (Not released yet)
+## 2.9.1
 
 ### `@liveblocks/client`
 
 - Fix type definition of `ThreadData`: `updatedAt` is always set
+- Fix bug where the old API endpoint was still being used for delta updates
```

Comment:
Which endpoint? I find the sentence a bit confusing, like there's a single endpoint in all `@liveblocks/client`. I would either specify which endpoint we're talking about or change it to "old API endpoints were"

Reply:
Thanks! I just updated the phrasing. Is this less confusing?

Reply:
Yep, thanks!